File: NetNodePluginClient/Request/ServerNetNodeRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using NetNodePluginClient.Model;
using EtherealC.Request.WebSocket;

namespace NetNodePluginClient.Request
{
    public class ServerNetNodeRequest:WebSocketRequest,IServerNetNodeRequest
    {
        public virtual NetNode GetNetNode(string servicename)
        {
            throw new NotImplementedException();
        }

        public override void Initializate()
        {

        }

        public override void UnInitialize()
        {

        }
    }
}

File: NetNodePluginClient/Request/ServerNetNodeRequest.py
from EtherealC.Net.NetNodeClient.Model.NetNode import NetNode
from EtherealC.Request.Decorator.Request import Request
from EtherealC.Request.WebSocket.WebSocketRequest import WebSocketRequest


class ServerNetNodeRequest(WebSocketRequest):
    def __init__(self, name, types):
        super().__init__()
        self.name = name
        self.types = types

    @Request()
    def GetNetNode(self, servicename: str) -> NetNode:
        pass
